src/ts/components/casttogglebutton.ts

```diff
@@ -14,6 +14,7 @@ export class CastToggleButton extends ToggleButton<ToggleButtonConfig> {
     this.config = this.mergeConfig(config, {
       cssClass: 'ui-casttogglebutton',
       text: i18n.getLocalizer('googleCast'),
+      ariaLabel: i18n.getLocalizer('googleCast'),
```

Comment:
Can't you set the `text` as default `ariaLabel`? Seems unnecessary to do this assignment over and over again.

Reply:
Unfortunately not, because in some cases the aria label is not the same as the text. E.g. `Play/Pause` is confusing for the users when it is shown on the button and the aria label `Play` would make more sense. When the button is active the user automatically implies that `Play` is active

Reply:
That's why I meant as default. Of course it can be overwritten where necessary.